test-runner/chrome/content/loggest-chrome-runner.js

```diff
@@ -1306,18 +1237,41 @@ function DOMLoaded() {
 
     try {
       runTests(configData).then(function(summaries) {
-        dump('\n\n***** ' + summaries.length + ' tests run: *****\n\n');
+        dump('\n************************\n\n');
+
         summaries.forEach(function(summary) {
           printTestSummary(summary);
         });
-        dump('\n************************\n\n');
+
+        dump('\n************************\n\n\n');
+        dump(' ╔══════════════════════════════════════════════════════════════╗\n');
+        dump(' ║ Run "make results" to view detailed results in your browser. ║\n');
+        dump(' ╚══════════════════════════════════════════════════════════════╝\n');
+        dump('                                                                 \n');
+        dump('           (Run "make clean" to delete old test logs.)           \n\n\n');
 
         var testResult = getTestResult(summaries);
         var jsonString = JSON.stringify({ result: testResult });
 
-        writeFile('test-logs', 'test-run.summary', jsonString).then(function() {
-          quitApp();
-        });
+        var INDEX_FILENAME = 'index.json';
+
+        readFile('test-logs', INDEX_FILENAME)
+          // Blah, file origins. Remove the JSONP function call before JSON-parsing.
```

Comment:
No, it's enabled.  Copy-and-paste for a nightly firefox on linux is "security.fileuri.strict_origin_policy;true" and it's shown as the default value.  Same deal on stable on Ubuntu.  Maybe things are different on OS X?  Or maybe you didn't blow away the symlinks/existing directories?  My changes to the Makefile were not enough on their own.